src/ducks/categories/helpers.js

```diff
@@ -75,6 +75,11 @@ export const transactionsByCategory = transactions => {
     // Creates a map of categories, where each entry contains a list of
     // related operations and a breakdown by sub-category
     const catId = getCategoryId(transaction)
+
+    if (!catId) {
```

Comment:
I would be prefer to have an explicity `if (isAwaitingCategory(catId))` here, it would be more readable IMHO.